webapp/package.json

```diff
@@ -99,6 +100,7 @@
       "text-summary"
     ],
     "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/src/$1",
```

Comment:
Why is this change needed?

Reply:
Jest was unable to resolve the imports after we changed the default import paths